generators/generator-microsoft-bot-adaptive/generators/app/index.js

```diff
@@ -21,6 +24,9 @@ const options = rt.Record({
 
 const defaultOptions = {
   applicationSettingsDirectory: undefined,
+  botProjectSettings: {
+    skills: undefined,
```

Comment:
Up to you, but I tend to just omit keys if the value is `undefined` as the end result is the same.